### PR TITLE
Add TikTok icon

### PR DESCRIPTION
**Pull request by as280093**
Closes issue #1375 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
note: first PR related to icons

color which i picked for background was black `#000000` as from original logo 
software used : Adobe Illustrator 

image

official tik tok icon contains 4 colors 
`#000000` `#66c2c9` `#e01a4e` `#ffffff`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2550,6 +2550,11 @@
             "hex": "000000",
             "source": "https://tidal.com"
         },
+        {
+            "title": "Tik Tok",
+            "hex": "000000",
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: The color for this one is not very clear 🤔 In general it's nice to shy away from black as it is a very generic color though this case I don't think it is necessarily wrong.

That said, looking at [the UI in the app](https://play.google.com/store/apps/details?id=com.zhiliaoapp.musically&referrer=af_tranid%3DULC8xI-j1M4RlbXYFLjVPw%26pid%3Dtiktokweb), I think the pink color (`#e01a4e`) may be a bit better as brand color.

Just my two cents, thoughts @as280093 and @kze?

**Comment by as280093** [comment]
ok i edit the pr as for now i am on my phone and for the color using black is my opinion as background of their official logo is black⚫

File: _data/simple-icons.json [hunk 2]
```diff
@@ -2550,6 +2550,11 @@
...
+            "source": "https://tiktok.com"
+        }
```
[the remark this answers is not in the document]
Comment (as280093) [reply _data/simple-icons.json]: fixed

File: _data/simple-icons.json (hunk 1 above)
Comment (ericcornelissen) [reply _data/simple-icons.json]: > ok i edit the pr as for now i am on my phone and for the color using black is my opinion as background of their official logo is black ⚫ | [in [this comment](https://github.com/simple-icons/simple-icons/pull/1376#issuecomment-486810673)]

Pinging @davidklebanoff for a third opinion

**Comment by as280093** [comment]
is this PR waiting for background suggestion?

**Comment by ericcornelissen** [comment]
> is this PR waiting for background suggestion?

For now it is yes 😄 I hope you don't mind waiting before this gets merged (I will ping @davidklebanoff again just in case)

File: _data/simple-icons.json (hunk 1 above)
Comment (ericcornelissen) [reply _data/simple-icons.json]: Repinging @davidklebanoff, also pinging @birjolaxew now

**Comment by ericcornelissen** [comment]
For lack of response, I decided to merge this PR. If it turns out we want to use a different color it can always be changed later.

Anyway, thanks for working on this @as280093 👌 My apologies for the long wait 😞

**Pull request merged**